app/main/views/frameworks.py

```diff
@@ -51,7 +51,7 @@ def framework_dashboard(framework_slug):
                 [user['emailAddress'] for user in supplier_users['users'] if user['active']],
                 email_body,
                 current_app.config['DM_MANDRILL_API_KEY'],
-                'You have started your {} application'.format(framework['name']),
+                'You started a {} application'.format(framework['name']),
```

Comment:
Good point.  I'm not sure what the motivation for changing it was... I'll check, would be nice not to set ourselves that trap for the future, but also nice to not build in logic we don't need right now.